syntax = "proto3";
option java_multiple_files = true;
package com.example.grpc.serverone.server;

message HelloRequest {
    string firstName = 1;
    string lastName = 2;
}

message HelloResponse {
    string greeting = 1;
}

service HelloServiceServerOne {
    rpc hello(HelloRequest) returns (HelloResponse);
}
